spec/plans/prepare.fmf

```diff
@@ -53,6 +53,9 @@ description: |
         Additionaly, the ``directory`` attribute can be used to
         install all packages from the given directory. Copr
         repositories can be enabled using the ``copr`` attribute.
+        Use the ``exclude`` option to skip selected packages from
+        installation.
```

Comment:
Maybe we should mention the options are directly passed to the package manager, thus support globbing ...

Reply:
Added an explicit mention about globbing.